### PR TITLE
Update project dashboard section

### PR DESCRIPTION
**Pull request by nterray**
Since Tuleap 9.9, project can have more than one dashboards.
Furthermore some paragraphs were not anymore accurate.
By the way update the screenshot :)

### CODE REVIEW

File: languages/en/user-guide/frs.rst [hunk 1]
```diff
@@ -102,8 +102,8 @@ administrators (see `Delivery Manager Structure`_).
 File Release Browsing and Download
 ----------------------------------
 
-The latest version of each project package (if any) is always visible on
-the Project Dashboard (see :ref:`an-example-of-project-dashboard`). The latest version can immediately be
+The latest version of each project package (if any) may be visible on
```
Comment (vaceletm) [inline languages/en/user-guide/frs.rst]: might be

File: languages/en/user-guide/project-summary.rst [hunk 2]
```diff
@@ -1,130 +1,58 @@
...
-   icon (|image1|) allows you to see the latest changes and developers
-   comments associated with this revision. Then comes the monitor icon
-   (|image2|). Selecting this icon will cause this package to be
-   monitored for you. Anytime the project development team posts a new
-   release, you will be automatically notified via e-mail. All monitored
-   File Releases are listed in your Personal Page and can be canceled
-   from this page or from the main page of the file release system.
+-  **Heartbeat**: the list of latest items updated in this project. This
+   is helpful to have an overview of the activity of the project. Please note
+   that not all services can push information in this widget.
```
Comment (vaceletm) [inline languages/en/user-guide/project-summary.rst]: As of today Artifacts and Git activities are listed. Permissions apply, it means that the content of hearbeat will change from one user to another according to their permissions.

File: languages/en/user-guide/frs.rst (hunk 1 above)
Comment (nterray) [inline languages/en/user-guide/frs.rst]: done

File: languages/en/user-guide/project-summary.rst (hunk 2 above)
Comment (nterray) [inline languages/en/user-guide/project-summary.rst]: done

**Pull request merged**